### PR TITLE
feat: render base data

### PR DESCRIPTION
**Pull request by zaida04**
This PR accomplishes our biggest task, loading submitted repository data into the website.
The data is stored in `./data/PROJECTS.json`, and on each commit to the repository the vercel page is rebuilt and populated with that data in a static manner.
This will most likely be refactored in the future to have an api so that we don't have to rebuild on every commit, this will save us build time for commits that don't even make any changes to the actual website data, like meta stuff.

sarthak rip this pr to shreds pls.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -47,29 +48,21 @@ export default function Home() {
...
-                        award="silver"
-                    />
-                    <Card
-                        name="Student Name"
-                        title="Lorem Ipsum"
-                        projectType="Website"
-                        thumbnail="https://source.unsplash.com/random"
-                        profileIcon="https://picsum.photos/200"
-                    />
+                    {RepoData.map((x) => {
```
Comment (sarthaktexas) [inline pages/index.js]: bad practices here, destructure each prop please
```suggestion
                    {RepoData.map(({repo_name, repo_owner, project_type, repo_logo, repo_owner_icon, award}) => {
```

File: pages/index.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { FiGithub } from "react-icons/fi";
 import Card from "../components/card";
 import Footer from "../components/footer";
 import HeadObject from "../components/head";
+import RepoData from "../data/PROJECTS.json";
```
Comment (sarthaktexas) [inline pages/index.js:8]: why in gods green world is this capitalized

File: data/CONTRIBUTORS.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[
```
Comment (sarthaktexas) [inline data/CONTRIBUTORS.json]: ya no clue why this file name is capitalized

Comment (sarthaktexas) [reply data/CONTRIBUTORS.json]: wait why does this file exist

Comment (sarthaktexas) [reply data/CONTRIBUTORS.json]: no no no no no just use the github api please so much easier

**Review by sarthaktexas — CHANGES_REQUESTED** [review]
fix pls

**Comment by sarthaktexas** [comment]
I want @awlevin's and @eilla1's input on the key names. I don't _love_ `repo_name` but i don't like `reponame` and `repoName` looks weird.  i'm fine with the current way but it looks wrong for some reason.

File: data/CONTRIBUTORS.json (hunk 3 above)
Comment (zaida04) [reply data/CONTRIBUTORS.json]: I thought we agreed to use dummy data until MVP stage passes and we can optimize stuff.

Comment (sarthaktexas) [reply data/CONTRIBUTORS.json]: i agree with dummy data for the projects but for the contributors?? unnecessary, it's just a simple fetch and render

File: pages/index.js (hunk 2 above)
Comment (zaida04) [reply pages/index.js:8]: So that I can remember to remove it later

**Comment by awlevin** [comment]
#### 📝Changed routes:


| [**/**](https://open-2m5cgtz1r-thinkfiveable.vercel.app/) |
|:-:|
|<a href="https://open-2m5cgtz1r-thinkfiveable.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/3MNdNSqpb23qeZ77BXo9vvUZ6ou5H81ksL3LMk-AI8r32pJZYcYvtl5I8TISIdjxoHpga-d35HqHhYbdRSM4wA.png">(view full size)</a> |







Commit a0bf9124dd9242c7a8bd2f3467e03c5c628c81fd (https://open-2m5cgtz1r-thinkfiveable.vercel.app).

File: pages/index.js (hunk 1 above)
Comment (zaida04) [reply pages/index.js]: resolved in 354279c

**Comment by zaida04** [comment]
https://github.com/ThinkFiveable/open/pull/41#issuecomment-816362173
resolved in 354279c

File: data/CONTRIBUTORS.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+        "github_username": "zaida04",
+        "github_icon": "https://avatars.githubusercontent.com/u/19440022?v=4"
+    },
+    {
+        "github_username": "eilla1",
+        "github_icon": "https://avatars.githubusercontent.com/u/72365100?v=4"
+    },
+    {
+        "github_username": "sarthaktexas",
+        "github_icon": "https://avatars.githubusercontent.com/u/28282096?v=4"
```
Comment (sarthaktexas) [inline data/CONTRIBUTORS.json]: you can get these by doing this:
```suggestion
        "github_icon": "https://avatars.githubusercontent.com/sarthaktexas"
```

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

File: data/CONTRIBUTORS.json (hunk 4 above)
Comment (sarthaktexas) [reply data/CONTRIBUTORS.json]: by the way this means you don't have to keep this key-pair at all

Comment (zaida04) [reply data/CONTRIBUTORS.json]: In that case, I can just remove that key and just template it.

**Comment by awlevin** [comment]
Lolol `repoName` looks fine to me

File: data/CONTRIBUTORS.json (hunk 3 above)
Comment (zaida04) [reply data/CONTRIBUTORS.json]: Fair point. I’ll resolve this tomorrow.

**Review by exu3 — APPROVED** [review]
looks good to me

File: pages/index.js [hunk 5]
```diff
@@ -47,29 +48,19 @@ export default function Home() {
...
+                            <Card
+                                key={repoName}
+                                name={repoOwner}
+                                title={repoName}
+                                projectType={projectType}
+                                thumbnail={repoLogo ?? "https://avatars.githubusercontent.com/u/76849512?v=4"}
+                                profileIcon={repoOwnerIcon}
+                                award={award ?? "silver"}
+                            />
+                        );
```
Comment (sarthaktexas) [inline pages/index.js:62]: did the linter really put a ; here???

Comment (zaida04) [reply pages/index.js:62]: Yeah? It's part of a return statement so ig i understand why it would

**Review by sarthaktexas — APPROVED** [review]

**Pull request merged**